src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -81,14 +83,18 @@ public function persist($object)
                 // Do nothing: not supported.
             }
 
-            try {
-                $this->objectManager->persist($object);
-            } catch (ORMException $exception) {
-                if ($metadata->idGenerator instanceof ORMAssignedGenerator) {
-                    throw ObjectGeneratorPersisterExceptionFactory::createForEntityMissingAssignedIdForField($object);
+            foreach ($this->managers as $manager) {
```

Comment:
Then, as you have an entity manager, you just need to do `$objectManager->persist($object)`.